packages/gatsby/src/commands/develop.js

```diff
@@ -258,6 +258,7 @@ async function startServer(program) {
 module.exports = async (program: any) => {
   initTracer(program.openTracingConfigFile)
   telemetry.trackCli(`DEVELOP_START`)
+  telemetry.startBackgroudUptate()
```

Comment:
is this also needed (or useful) in `gatsby serve` which is second non-self-exiting gatsby command?

Reply:
Indeed, we should add it for that too. 
My Mac is just installing updates so I'll update this tomorrow morning. Or if we want to ship this before it, feel free to edit the pr :)

Reply:
Done, shipping now